taxonomy-editor-frontend/src/components/ResponsiveAppBar.tsx

```diff
@@ -126,7 +126,7 @@ const ResponsiveAppBar = ({ displayedPages }: ResponsiveAppBarProps) => {
             >
               <MuiLink
                 sx={{ mr: 2, display: "flex", alignSelf: "center" }}
-                href="https://world.openfoodfacts.org/"
+                href="https://ui.taxonomy.openfoodfacts.net/"
                 target="_blank"
               >
```

Comment:
This should probably be a `react-router-dom` `Link` to the `/` path (you can look at how it is done below with the `Taxonomy Editor` text), as the link we want the logo to poin to depends on where the app is deployed